packages/schematics/angular/application/files/standalone-files/src/main.ts.template

```diff
@@ -0,0 +1,7 @@
+import { bootstrapApplication } from '@angular/platform-browser';
+import { appConfig } from './app/app.config';
+import { AppComponent } from './app/app.component';
+
+bootstrapApplication(AppComponent, appConfig).catch((err) =>
```

Comment:
Quick question: should we use uppercase names here and for the `ROUTES` above?
```suggestion
import { CONFIG } from './app/app.config';
import { AppComponent } from './app/app.component';

bootstrapApplication(AppComponent, CONFIG).catch((err) =>
```

Reply:
I would use the lower-case names here, since I don't think of these things as _constants_